### PR TITLE
Add missing/new tests, remove upgrade test as it is covered by others.

### PR DESCRIPTION
**Pull request by seancribbs**

### CODE REVIEW

File: db/seed.rb [hunk 1]
```diff
@@ -24,20 +24,25 @@
 }
 
 riak_tests = %w{
+  basic_command_line
+  client_java_verify
+  client_ruby_verify
```
Comment (joedevivo) [inline db/seed.rb]: ruby test requires memory backend, so will need special handling, like secondary index tests.

File: db/seed.rb [hunk 2]
```diff
@@ -24,20 +24,25 @@
...
   gh_riak_core_154
   gh_riak_core_155
   gh_riak_core_176
+  loaded_upgrade
```
Comment (joedevivo) [inline db/seed.rb]: As much as it pains me to say it, we're going to want this for bolth Sir Bitifred VonCask and Levelenstein D Brown.

**Comment by cmeiklejohn** [comment]
We'll need to manually remove those upgrade tests as well -- migration?

**Comment by seancribbs** [comment]
Thanks for the comments, will fix.

**Comment by seancribbs** [comment]
@cmeiklejohn @joedevivo another look, please?

File: db/schema.rb [hunk 3]
```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended to check this file into your version control system.
```
Comment (joedevivo) [inline db/schema.rb:12]: Well done then! :P

**Comment by joedevivo** [comment]
+1

**Pull request merged**